src/Service/CurriculumInventory/VerificationPreviewBuilder.php

```diff
@@ -100,13 +100,16 @@ class VerificationPreviewBuilder
      * CurriculumInventoryVerificationReportBuilder constructor.
      *
      * @param Aggregator $aggregator
-     * @param AamcMethodManager $methodManager
-     * @param AamcPcrsManager $pcrsManager
+     * @param AamcMethodRepository $methodManager
+     * @param AamcPcrsRepository $pcrsManager
      */
-    public function __construct(Aggregator $aggregator, AamcMethodManager $methodManager, AamcPcrsManager $pcrsManager)
-    {
+    public function __construct(
+        Aggregator $aggregator,
+        AamcMethodRepository $methodManager,
+        AamcPcrsRepository $pcrsManager
```

Comment:
variable name should be `$pcrsRepository` as well.

Reply:
✔️